custom-metrics-stackdriver-adapter/adapter.go

```diff
@@ -211,7 +229,13 @@ func main() {
 		"Stackdriver Endpoint used by adapter. Default is https://monitoring.googleapis.com/")
 	flags.BoolVar(&serverOptions.EnableDistributionSupport, "enable-distribution-support", serverOptions.EnableDistributionSupport,
 		"enables support for scaling based on distribution values")
-
+	// Custom flags
+	flags.StringVar(&serverOptions.GcpProject, "project-id", "",
+		"Google Cloud Platform Project")
+	flags.StringVar(&serverOptions.GcpCluster, "cluster", "",
+		"Google Cloud Platform Cluster")
+	flags.StringVar(&serverOptions.GcpLocation, "location", "",
+		"Google Cloud Platform Location")
```

Comment:
Could you add some more informative descriptions for these flags? Specifically do they refer to where your stackdriver instance is running? Does `cluster` mean the cluster you are running on (so aws-inference-3 for us)?

Reply:
Ok, I copied and slightly clarified the descriptions from the Prometheus Operator that we're also using. https://github.com/GoogleCloudPlatform/prometheus-engine/blob/main/cmd/operator/main.go#L63-L65